src/xdocs/config_misc.xml

```diff
@@ -271,7 +271,8 @@ String unitAbbrev = "\u03bc\u03bc\u03bc";
 &lt;/module&gt;
         </source>
         <p>
-          An example of how to configure the check to allow non-printable escapes:
+          An example of how to configure the check to allow using escapes
+          for non-printable whitespace characters:
```

Comment:
`non-printable whitespace` => `non-printable, whitespace`

Reply:
done. Fixed it wherever I found it.